src/pages/notifications/index.vue

```diff
@@ -297,6 +298,23 @@ export default {
       return this.convertToGroupedEvents(this.processedEvents);
     },
   },
+  watch: {
+    getLatestEventTimestamp: {
+      immediate: true,
+      handler(newVal) {
+        if (newVal) {
+          try {
+            this.$api.$post(updateEventLastSeen(), {
+              ts: Number(newVal),
+            });
+            this.lastUpdatedTime = newVal;
```

Comment:
Does this trigger if I don't refresh the events at all? i.e. when is `this.$api.$post(updateEventLastSeen()` triggered if i don't refresh?